src/settings_type.h

```diff
@@ -222,6 +222,8 @@ struct GUISettings {
 
 	bool   scale_bevels;                     ///< bevels are scaled with GUI scale.
 
+	std::string digit_decimal_separator; ///< decimal separator
```

Comment:
I have no idea why this is or was a setting. I would remove it :)

Is this for debugging?

Reply:
It's so someone can override whatever their chosen language dictates. Just like the format and abbreviation configuration.